package.json

```diff
@@ -47,6 +47,7 @@
     "classnames": "^2.2.5",
     "common-tags": "^1.7.2",
     "copy-to-clipboard": "^3.0.8",
+    "copy-webpack-plugin": "^4.5.1",
```

Comment:
I think this should be in devDependencies

Reply:
Done.